Example/Backpack/ViewControllers/SwiftUIHostViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2022 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import SwiftUI
+
+class SwiftUIHostViewController<Content: View>: UIViewController {
+    
+    public var content: Content?
```

Comment:
can we make the contentView `not optional` and receive the `content` on the initialiser as `non optional`? by how the class work, it doesn't seem to be conceptually optional.
it will ensure there's content to be setup in the viewDidLoad, to avoid temporal coupling.
It can probably be a `let` too
also no need to store the content instance.

```swift
class ContentUIHostingController<Content: View>: UIViewController {
  private let contentView: UIHostingController<Content>

  init(_ content: Content) { 
    contentView = UIHostingController(rootView: content)
    super.init()
  }
  required init?(coder: NSCoder) {
    fatalError("init(coder:) has not been implemented")
  }
  
  override func viewDidLoad() {
    super.viewDidLoad()
    addChild(contentView)
    view.addSubview(contentView.view)
    setupConstraints()
  }
  private func setupConstraints() { ... }
}
// And use it like ...
ContentUIHostingController(Text("Hello!))
```